os-java-run/added/java-default-options

```diff
@@ -83,7 +83,13 @@ initial_memory() {
     if [ "${ms}" -lt "${max_initial_memory}" ] ; then
       echo "-Xms${ms}m"
     else
-      echo "-Xms${max_initial_memory}m"
+      # check if JAVA_MAX_INITIAL_MEM is unset. If it is we need to check if ms > max_initial_memory and use the larger value
+      # instead of the default
+      if [ "x${JAVA_MAX_INITIAL_MEM}" = "x" ] && [ "${ms}" -gt "${max_initial_memory}" ]; then
```

Comment:
This seems counter intuitive to me, since we have a default setting for initial mem.  Should we just do away with the default?

Reply:
Also, the default is 4 gig, which seems pretty high to me.

Reply:
Based on the jira, it sounds like we should modify the default to be container max mem, then bound min by either max mem or container max mem.